notifications_utils/logging.py

```diff
@@ -17,6 +18,41 @@
 
 logger = logging.getLogger(__name__)
 
+_phone_regex = re.compile("(?:\\+ *)?\\d[\\d\\- ]{7,}\\d")
+
+
+def _scrub(msg: Any) -> Any:
+    # Sometimes just an exception object is passed in for the message, skip those.
+    if not isinstance(msg, str):
+        return msg
+    phones = _phone_regex.findall(msg)
+
+    phones = [phone.replace("-", "").replace(" ", "") for phone in phones]
+    for phone in phones:
+        msg = msg.replace(phone, "1XXXXXXXXXX")
+
+    emails = re.findall(
+        r"[\w\.-]+@[\w\.-]+", msg
```

Comment:
D'oh, sorry, meant to include this regex along with the phone number for `re.compile`!